.github/workflows/run_tests.yaml

```diff
@@ -16,3 +16,34 @@ jobs:
         docker-compose build
         docker-compose run re_api sh scripts/run_tests.sh
         docker-compose down --remove-orphans
+
+  docker_build_and_push:
+    runs-on: ubuntu-latest
+    needs: run_tests
+    if: (github.ref == 'refs/heads/develop' || github.ref == 'refs/heads/master') && github.event_name == 'push' && !contains(github.event.head_commit.message, 'skip_docker_build')
```

Comment:
runs on pushes (i.e. merges) to `develop` or `master`
to skip the upload, include `skip_docker_build` in the commit message

Reply:
nice